admin/workspace-providers/deployment.md

```diff
@@ -106,7 +108,8 @@ when communicating with the Coder deployment.
       --install \
       --force \
```

Comment:
since we no longer use `--force` internally, should we remove here too? maybe @sreya has an opinion on that

Reply:
Yep, should remove `--force` from all docs. @bpmct removed a couple here already https://github.com/cdr/docs/pull/171